Etabs_Adapter/CRUD/Update/Node.cs

```diff
@@ -27,6 +27,9 @@
 using BH.oM.Structure.Elements;
 using BH.oM.Structure.Constraints;
 using BH.Engine.Adapters.ETABS;
+using System.Collections;
+using System.Xml.Linq;
```

Comment:
Is this needed?

Reply:
@peterjamesnugent, thanks for the comment.
Libraries not used. References deleted from the code in latest commit